app/controllers/simple.coffee

```diff
@@ -36,3 +36,14 @@ App.EmberTableSimpleController = Ember.Controller.extend
       close: Math.random() * 100 - 50
       volume: Math.random() * 1000000
   .property 'numRows'
+
+  actions:
```

Comment:
we shouldn't add this to a simple example, and probably shouldn't change the demo app in general.